.docker/app/Dockerfile

```diff
@@ -1,10 +1,24 @@
 ARG NEXTCLOUD_VERSION=stable-fpm
 
+
+
 FROM nextcloud:${NEXTCLOUD_VERSION}
 
 ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
 RUN chmod uga+x /usr/local/bin/install-php-extensions && sync \
     && install-php-extensions \
     bz2
 
-COPY config/php.ini /usr/local/etc/php/conf.d/
+ENV PHP_PM_MAX_CHILDREN=10
+ENV PHP_PM_START_SERVERS=2
+ENV PHP_PM_MIN_SPARE_SERVERS=1
+ENV PHP_PM_MAX_SPARE_SERVERS=3
+    
+RUN set -eux; \
+    echo '[www]' > /usr/local/etc/php-fpm.d/zz-pm.conf; \
+    echo "pm.max_children = ${PHP_PM_MAX_CHILDREN}" >> /usr/local/etc/php-fpm.d/zz-pm.conf; \
+    echo "pm.start_servers = ${PHP_PM_START_SERVERS}" >> /usr/local/etc/php-fpm.d/zz-pm.conf; \
+    echo "pm.min_spare_servers = ${PHP_PM_MIN_SPARE_SERVERS}" >> /usr/local/etc/php-fpm.d/zz-pm.conf; \
+    echo "pm.max_spare_servers = ${PHP_PM_MAX_SPARE_SERVERS}" >> /usr/local/etc/php-fpm.d/zz-pm.conf
```

Comment:
These environments could be added by volumes
By example, we can add this volume:
```yaml
volumes:
 - volumes/php/pm.ini:/usr/local/etc/php/conf.d/
```

And put the follow content to `volumes/php/pm.ini` file:
```ini
[www]
pm.max_children = 10
pm.start_servers = 2
pm.min_spare_servers = 1
pm.max_spare_servers = 3
```

Doing this, dont need to do any change  at Dockerfile.

Maybe we can put this as suggestion at documentation of this repository explaining about how to set best values to these settings.